install.py

```diff
@@ -0,0 +1,150 @@
+import os
+from zipfile import ZipFile
+
+IGUASSU_BRANCH=
+ARREBOL_BRANCH=
+WEB_UI_BRANCH=
+
+#Properties names
+oauth_storage_service_token_url="oauth_storage_service_token_url"
+oauth_storage_service_client_id="oauth_storage_service_client_id"
+oauth_storage_service_client_secret="oauth_storage_service_client_secret"
+
+storage_service_host_url="storage_service_host_url"
+arrebol_service_host_url="arrebol_service_host_url"
+iguassu_service_host_url="iguassu_service_host_url"
+web_ui_host_url="web_ui_host_url"
+
+job_state_monitor_period="job_state_monitor_period"
+session_monitor_period="session_monitor_period"
+job_submission_monitor_period="job_submission_monitor_period"
+
+VUE_APP_EXTERNAL_LINK_DOCS="VUE_APP_EXTERNAL_LINK_DOCS"
+VUE_APP_EXTERNAL_LINK_GITHUB_REPO="VUE_APP_EXTERNAL_LINK_GITHUB_REPO"
+VUE_APP_EXTERNAL_LINK_ABOUT="VUE_APP_EXTERNAL_LINK_ABOUT"
+
+VUE_APP_OWNCLOUD_SERVER_HOST="VUE_APP_OWNCLOUD_SERVER_HOST"
+VUE_APP_OWNCLOUD_OAUTH_CLIENT_ID="VUE_APP_OWNCLOUD_OAUTH_CLIENT_ID"
+VUE_APP_OWNCLOUD_OAUTH_SECRET="VUE_APP_OWNCLOUD_OAUTH_SECRET"
+VUE_APP_OWNCLOUD_OAUTH_REDIRECT_URI="VUE_APP_OWNCLOUD_OAUTH_REDIRECT_URI"
```

Comment:
Note that this conf property is the same as set by OAuth2 to `redirect_uri`. In this case, replace the name of this property and group it together with the other OAuth2 properties.